Link_sinFade_Library.ino

```diff
@@ -32,10 +31,12 @@ void HSV_sinFade_Retain_1Z (float Steps, int fadeZone, float startHSV[], float H
   for (int Stepr = 0; Stepr <= Steps; Stepr++) {    ///////////// <<<<<<<<<<<<  START FADE
 
     if (! (Stepr % 10) ) {          // modulo operator % returns the remainder, e.g.: 10 % 2 = 0. 10 % 3 = 1.
+      // FIXME: should this be a map()? looks like a 0..1
       ValKnob = analogRead(BRIGHTNESS) / 1043.9 + 0.02;            // analogRead is slow, so do every 10th
```

Comment:
you are mapping from 0..1024 to 0.02..1043 ?
Probably because the pot is not giving 0..1024, so this is the correction from experiment?